developer_manual/basics/middlewares.rst

```diff
@@ -4,7 +4,7 @@ Middlewares
 
 .. sectionauthor:: Bernhard Posselt <dev@bernhard-posselt.com>
 
-Middleware is logic that is run before and after each request and is modelled after `Django's Middleware system <https://docs.djangoproject.com/en/dev/topics/http/middleware/>`_. It offers the following hooks:
```

Comment:
That is only in US English: https://www.oxfordlearnersdictionaries.com/definition/english/model_2

Reply:
Good catch! My Grammarly is set to American English indeed.
So the docs should be based on British English?

Reply:
Good question. I think that has never been defined explicitly.

Reply:
Maybe it can be added to the to-do list of the PR template?
Either way and regardless of personal preferences, I'm always in favor of consistency.

Reply:
Would be good to clear this up. For this backport it's probably too late and the same change is already in the master branch. So let's finish this here.